pkg/extract/extract.go

```diff
@@ -24,6 +24,7 @@ const minSizeForProgressBar = 100_000_000
 type Options struct {
 	ShowProgress bool
 	FileFilter   func(string) bool
+	Flatten      bool
```

Comment:
@guillaumerose I am not able to get why this is required and what is the use of this (as of now I don't see it in used excepted the testing part). Can you please add bit description on the commit log around same?

Reply:
yes I simplified it. I will bring the bundle name in the future more globally.